src/main/java/org/traccar/config/Keys.java

```diff
@@ -1162,6 +1162,27 @@ private Keys() {
             "notificator.telegram.sendLocation",
             List.of(KeyType.CONFIG));
 
+    /**
+     * Viber notification API key.
+     */
+    public static final ConfigKey<String> NOTIFICATOR_VIBER_KEY = new StringConfigKey(
+                    "notificator.viber.key",
+                    List.of(KeyType.CONFIG));
+
+    /**
+     * Viber notification chat id to post messages to.
+     */
+    public static final ConfigKey<String> NOTIFICATOR_VIBER_CHAT_ID = new StringConfigKey(
+                    "notificator.viber.uid",
+                    List.of(KeyType.CONFIG));
+
+    /**
+     * Viber notification send location message.
+     */
+    public static final ConfigKey<Boolean> NOTIFICATOR_VIBER_SEND_LOCATION = new BooleanConfigKey(
```

Comment:
We should probably unify this config parameter between telegram and viber.

Reply:
What would we name this unified parameter ? Is NOTIFICATOR.SENDLOCATION = "notificator.sendLocation" okey ?

Reply:
I think we may keep this as it is for now to ensure backward compatibility, changing the parameter name would require modifications when upgrading from an older version for anyone using this features. What do you think ?

Reply:
`NOTIFICATOR_SEND_LOCATION = "notificator.sendLocation"` sounds good to me. I think it's a rarely used key, so it's ok to break backward compatibility. Also, it doesn't really break notifications. You just won't get the location info.